packages/@aws-cdk/aws-lambda/package.json

```
@@ -127,7 +128,8 @@
     "@aws-cdk/aws-sqs": "0.0.0",
     "@aws-cdk/core": "0.0.0",
     "@aws-cdk/cx-api": "0.0.0",
-    "constructs": "^3.3.69"
+    "constructs": "^3.3.69",
+    "@aws-cdk/region-info": "0.0.0"
```

minor: we usually like to keep this alphabetically sorted

```suggestion
    "@aws-cdk/region-info": "0.0.0",
    "constructs": "^3.3.69"
```